test/e2e/scenario/recorder/recorder.scenario.ts

```diff
@@ -806,6 +806,34 @@ describe('recorder', () => {
       })
   })
 
+  describe('recording of sampled out sessions', () => {
+    createTest('should not start recording when session is sampled out')
+      .withRum({ sessionReplaySampleRate: 0 })
+      .withSetup(bundleSetup)
+      .run(async ({ intakeRegistry }) => {
+        await browser.execute(() => {
+          window.DD_RUM!.startSessionReplayRecording()
+        })
+
+        await flushEvents()
+
+        expect(intakeRegistry.replaySegments.length).toBe(0)
+      })
+
+    createTest('should start recording if forced when session is sampled out')
+      .withRum({ sessionReplaySampleRate: 0 })
+      .withSetup(bundleSetup)
+      .run(async ({ intakeRegistry }) => {
+        await browser.execute(() => {
+          window.DD_RUM!.startSessionReplayRecording({ force: true })
+        })
+
+        await flushEvents()
+
+        expect(intakeRegistry.replaySegments.length).toBe(1)
```

Comment:
**💬 suggestion:** ‏same here, you could add an assertion to ensure `forceReplay=1` is in the cookie.

Reply:
Done ✔️